tests/src/Kernel/fixtures/markup_rendering_patterns/progress.yml

```diff
@@ -37,7 +37,7 @@ progress_secondary_non_stripped_non_animated:
     '#id': progress
     '#fields':
       settings:
-        style: 'secondary'
+        style: 'light'
```

Comment:
What's the goal here? You want to test that the text color changes if you switch to 'light'?
And we no longer need to test 'secondary'?

I would say:
- If we still want to test 'secondary' we should instead add another scenario to test with 'light'.
- If we want to drop the 'secondary' scenario and only test 'light', we should rename the scenario. E.g. 'progress_light_non_stripped_non_animated'.

I think there is probably nothing special about 'secondary', so probably the second option is ok.